zkb

```diff
@@ -1,9 +1,20 @@
 #!/bin/bash
 # Files that refer to this file by ID. (Backlinks)
 
+ZKPATH="${ZKPATH:-~/Dropbox/wiki/archive}"
+
+if [ ! -d "$ZKPATH" ]; then
+  echo "Zettelkasten path \"$ZKPATH\" does not exist."
+  exit 1
+fi
```

Comment:
We totally could. It would require all of these files to be in the same folder. 

I toyed with the idea of moving all of these to the same script file since they're so short and share quite a lot. You can export each command as functions. Or we could do (overloaded) subcommands:

```
zk b
zk backlinks
```